srcpkgs/dpdk/template

```diff
@@ -14,12 +15,19 @@ homepage="https://www.dpdk.org/"
 distfiles="https://github.com/DPDK/${pkgname}/archive/v${version}.tar.gz"
 checksum=1ceff1a6f4f8d5f6f62c1682097249227ac5225ccd9638e0af09f5411c681038
 
-if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
-	broken="Usage of error.h, execinfo.h and argp.h"
-fi
-
-case "$XBPS_TARGET_MACHINE" in
-	armv*) broken="Cannot get define of '__ARM_NEON'";;
+case "${XBPS_TARGET_MACHINE}" in
+	*-musl)
+		broken="Usage of error.h, execinfo.h and argp.h"
+	;;
+	armv6l)
+		configure_args+=" -Dmachine=armv6-a"
+	;;
+	armv7l)
+		configure_args+=" -Dmachine=armv7-a"
+	;;
+	aarch64)
+		configure_args+=" -Dmachine=armv8-a"
+	;;
```

Comment:
also add a new case

```
ppc64le*) configure_args+=" -Dmachine=power8`;;
```

you can also turn these case statements into single lines, as long as they fit in 80col